core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/redhat-7/add-repositories-noarch.sh

```diff
@@ -0,0 +1,120 @@
+gpgcheck=1
+gpgkey=https://artifacts.elastic.co/GPG-KEY-elasticsearch
+enabled=1
+autorefresh=1
+type=rpm-md
+EOF
+)
+
+ELASTICSEARCH_CURATOR_REPO_CONF=$(cat <<'EOF'
+[curator-5]
+name=CentOS/RHEL 7 repository for Elasticsearch Curator 5.x packages
+baseurl=https://packages.elastic.co/curator/5/centos/7
+gpgcheck=1
+gpgkey=https://packages.elastic.co/GPG-KEY-elasticsearch
+enabled=1
+EOF
+)
+
+GRAFANA_REPO_CONF=$(cat <<'EOF'
+[grafana]
+name=grafana
+baseurl=https://packages.grafana.com/oss/rpm
+repo_gpgcheck=1
+enabled=1
+gpgcheck=1
+gpgkey=https://packages.grafana.com/gpg.key
+sslverify=1
+sslcacert=/etc/pki/tls/certs/ca-bundle.crt
+EOF
+)
+
+KUBERNETES_REPO_CONF=$(cat <<'EOF'
+[kubernetes]
+name=Kubernetes
+baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64
+enabled=1
+gpgcheck=1
+repo_gpgcheck=1
+gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+EOF
+)
+
+OPENDISTRO_REPO_CONF=$(cat <<'EOF'
+[opendistroforelasticsearch-artifacts-repo]
+name=Release RPM artifacts of OpenDistroForElasticsearch
+baseurl=https://d3g5vo6xdbdb9a.cloudfront.net/yum/noarch/
+enabled=1
+gpgkey=https://d3g5vo6xdbdb9a.cloudfront.net/GPG-KEY-opendistroforelasticsearch
+gpgcheck=1
+repo_gpgcheck=1
+autorefresh=1
+type=rpm-md
+EOF
+)
+
+POSTGRESQL_REPO_CONF=$(cat <<'EOF'
+[pgdg10]
+name=PostgreSQL 10 for RHEL/CentOS $releasever - $basearch
+baseurl=https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-$releasever-$basearch
+enabled=1
+gpgcheck=1
+gpgkey=https://download.postgresql.org/pub/repos/yum/RPM-GPG-KEY-PGDG
+EOF
+)
+
+RABBITMQ_SERVER_REPO_CONF=$(cat <<'EOF'
+[bintray-rabbitmq-server]
+name=bintray-rabbitmq-rpm
+baseurl=https://dl.bintray.com/rabbitmq/rpm/rabbitmq-server/v3.8.x/el/7/
+gpgcheck=1
+gpgkey=https://dl.bintray.com/rabbitmq/Keys/rabbitmq-release-signing-key.asc
+repo_gpgcheck=1
+enabled=1
+EOF
+)
+
+# Official Docker CE repository, added with https://download.docker.com/linux/centos/docker-ce.repo,
+# has broken URL (https://download.docker.com/linux/centos/7Server/x86_64/stable) for longer time.
+# So direct (patched) link is used first if available.
+add_repo_as_file 'docker-ce-stable-patched' "$DOCKER_CE_PATCHED_REPO_CONF"
+if ! is_repo_available "docker-ce-stable-patched"; then
+	disable_repo "docker-ce-stable-patched"
+	add_repo 'docker-ce' 'https://download.docker.com/linux/centos/docker-ce.repo'
+fi
+add_repo_as_file 'elastic-6' "$ELASTIC_6_REPO_CONF"
+add_repo_as_file 'elasticsearch-7' "$ELASTICSEARCH_7_REPO_CONF"
+add_repo_as_file 'elasticsearch-curator-5' "$ELASTICSEARCH_CURATOR_REPO_CONF"
+add_repo_as_file 'grafana' "$GRAFANA_REPO_CONF"
+add_repo_as_file 'kubernetes' "$KUBERNETES_REPO_CONF"
+add_repo_as_file 'opendistroforelasticsearch' "$OPENDISTRO_REPO_CONF"
+add_repo_as_file 'postgresql-10' "$POSTGRESQL_REPO_CONF"
+add_repo_as_file 'bintray-rabbitmq-rpm' "$RABBITMQ_SERVER_REPO_CONF"
```

Comment:
\n

Reply:
Fixed \n. And yeah weill merge develop into the ARM branch after 0.10.0 release to add the repo and other changes.